tests/containers/podman_bci_systemd.pm

```diff
@@ -23,6 +23,9 @@ sub run {
     record_info('Test', 'Launch a container with systemd');
     assert_script_run("podman run -d -p 80:80 --name nginx $image");
 
+    # Wait for the container to be up
+    script_retry("podman exec nginx cat /etc/os-release", retry => 10, delay => 15);
```

Comment:
It would work but it's a bit bulky.

How about using `podman wait --condition=healthy nginx` and add a `--healthcheck="curl http://localhost"` in the `podman run` command before?

Reply:
I am going to test it.
However the last VR in opensuse failed

Reply:
wait condition is not in the current version of podman. I did an other solution